src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1292,6 +1291,64 @@ public void ShowUpdatePackages(ShowUpdatePackageOptions updatePackageOptions)
                     SelectMatchingUpdatePackages(updatePackageOptions);
                 };
                 _packageList.LoadItemsCompleted += handler;
+            }  
+        }
+
+        public void SelectFirstPackage(string packageVersion)
```

Comment:
SelectFirstPackage should handle the case where the list is already loaded so it selects the first package and only needs to hook up an event handler to wait for the list to load if the list is not already loaded. That will make this method more general purpose and reflective of its name.